ci/build-msys2.sh

```diff
@@ -7,20 +7,21 @@ meson setup build            \
   -D d3d-hwaccel=enabled     \
   -D d3d11=enabled           \
   -D dvdnav=enabled          \
-  -D egl-angle-lib=enabled   \
-  -D egl-angle-win32=enabled \
   -D jpeg=enabled            \
   -D lcms2=enabled           \
   -D libarchive=enabled      \
   -D libbluray=enabled       \
   -D libmpv=true             \
   -D lua=enabled             \
-  -D pdf-build=enabled       \
-  -D rubberband=enabled      \
   -D shaderc=enabled         \
   -D spirv-cross=enabled     \
   -D tests=true              \
   -D uchardet=enabled        \
   -D vapoursynth=enabled
+
+if [[ "$SYS" != "clang32" && "$SYS" != "mingw32" ]]; then
+   meson configure build -D{egl-angle-lib,egl-angle-win32,pdf-build,rubberband}=enabled
+fi
```

Comment:
doesn't msys have bash? I think this is nicer:
```bash
args=(
  --werror
  -Dfoo=enabled
)
something && args+=(-Dbar=enabled)
meson configure build "${args[@]}"
```

Reply:
I've added commit to compact the script, take a look now, if it is ok.